tools/cvm-image-rewriter/run.sh

```diff
@@ -41,15 +36,36 @@ Optional
                             if install libvirt, you can specify to "/var/run/libvirt/libvirt-sock"
                             then the corresponding URI is "qemu+unix:///system?socket=/var/run/libvirt/libvirt-sock"
   -n                        Silence running for virt-install, no output
-  -h                        Show usage
 
 EOM
 }
 
+#
+# Update plugin directory list
+#
+# If want to skip some steps, please create a file named "NOT_RUN"
```

Comment:
I would reword  "To skip the installation of a specific plugin place a file called "NOT_RUN" in each plugin directory to be skipped"